resources/zib/terminology/ValueSet-GCS_EyesCodelijst-2.16.840.1.113883.2.4.3.11.60.40.2.12.8.1--20200901000000.xml

```diff
@@ -1,12 +1,21 @@
 <ValueSet xmlns="http://hl7.org/fhir">
    <id value="2.16.840.1.113883.2.4.3.11.60.40.2.12.8.1--20200901000000"/>
    <meta>
+      <source value="http://decor.nictiz.nl/fhir/4.0/zib2020bbr-"/>
       <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset"/>
       <!--<profile value="http://hl7.org/fhir/4.0/StructureDefinition/ValueSet"/>-->
+      <tag>
```

Comment:
Why do these ValueSet get a tag and a meta.source?
Include in release notes and add to all?

Reply:
Because ART-DECOR has found a good reason to add them. Let's not touch generated materials if not needed.